src/main/java/com/client/core/resttrigger/controller/jobsubmission/JobSubmissionRestTriggerController.java

```diff
@@ -17,67 +21,95 @@
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.ResponseBody;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Controller
 @RequestMapping("/resttrigger/jobsubmission/*")
 public class JobSubmissionRestTriggerController extends AbstractRestTriggerController<JobSubmission, JobSubmissionRestTriggerHelper, JobSubmissionRestTriggerTraverser> {
 
     private final Logger log = Logger.getLogger(JobSubmissionRestTriggerController.class);
+    private final JsonConverter jsonConverter;
 
     @Autowired
-    public JobSubmissionRestTriggerController(Optional<List<TriggerValidator<JobSubmission, JobSubmissionRestTriggerHelper, JobSubmissionRestTriggerTraverser>>> triggerValidators) {
+    public JobSubmissionRestTriggerController(Optional<List<TriggerValidator<JobSubmission, JobSubmissionRestTriggerHelper, JobSubmissionRestTriggerTraverser>>> triggerValidators, JsonConverter jsonConverter) {
         super(JobSubmission.class, triggerValidators, JobSubmissionRelatedEntity.values());
+        this.jsonConverter = AppContext.getApplicationContext().getBean(JsonConverter.class);
```

Comment:
we should just add this to the constructor, i.e. 'JsonConverter jsonConverter' as a parameter after the Optional<List> as opposed to calling on AppContext